bin/millau-runtime/src/lib.rs

```diff
@@ -0,0 +1,475 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
```

Comment:
I wonder if instead of duplicating the runtime code it makes more sense to have it conditionally compiled? Also couldn't we just do with a different genesis state?

Reply:
I don't know if we can quite get everything conditionally compiled properly. From what I remember some macros, like `construct_runtime!`, don't support conditional compilation within them